Exercise-4/isPrime.js

```diff
@@ -0,0 +1,29 @@
+// A function that checks if a given number is prime or not
+
+const isPrime = (number) => {
+  // Check if number is a real number
+  if (typeof number !== "number") return false;
+
+  // Check factor 1: The smallest number that satisfies the definition of prime numbers is 2
+  if (number < 2) {
+    return false;
+  }
+
+  // Check factor2 : The number not be divisible by any other number except 1 and itself
+  for (let i = 2; i < Math.ceil(Math.sqrt(number)); i++) {
```

Comment:
please check the condition again, because it produces unexpected results.
eg: isPrime(4) // true